Client/src/Components/Sidebar/index.jsx

```diff
@@ -173,7 +173,7 @@ function Sidebar() {
       }}
     >
       <Stack pt={theme.spacing(6)} pb={theme.spacing(12)} pl={theme.spacing(8)}>
-        <Stack direction="row" alignItems="center" gap={theme.spacing(4)}>
+        <Stack direction="row" alignItems="center" gap={theme.spacing(4)} onClick={() => window.open("https://github.com/bluewave-labs/bluewave-uptime", "_blank", "noreferrer")}>
```

Comment:
Can you add `sx={{ cursor: "pointer" }}` to this stack please?  That's the MUI way of declaring an inline style.

Thanks!